Classes/Private/Manager/Video/OCTVideoEngine.h

```diff
@@ -56,16 +47,16 @@
 - (BOOL)isSendingVideo;
 
 /**
- * Current video feed.
+ * Generate a UIView with the current incoming video feed.
  */
-- (UIView *)videoFeedWithRect:(CGRect)rect;
+- (UIView *)videoFeed;
 
 /**
  * Layer of the preview video.
  * @return CALayer that can be added to a UIView. Nil if
  * OCTVideoEngine is not sending any video.
  */
-- (CALayer *)videoCallPreview;
+- (void)getVideoCallPreview:(void (^)(CALayer *layer))completionBlock;
```

Comment:
Can you please add documentation that completionBlock is a required parameter? You can also add an NSAssert for that